File: TD/TD3/Code/3.cpp
/**
 * Afficher les dix nombres suivants la valeur N donnée en paramètre
 */
#include <iostream>

void affiche(int a)
{
  for (int i = 0; i < 10; ++i) {
    std::cout << a << " " << std::flush;
    ++a;
  }
}

int main()
{
  int a;
  
  std::cout << "Donne moi une valeur : " << std::flush;
  std::cin >> a;
  
  affiche(a);
  
  std::cout << std::endl;
  
  return 0;
}

File: TD/TD3/Code/Python/3.py
#!/usr/bin/python
# Afficher les dix nombres suivants la valeur N donnée en paramètre


def show_num(a):
    for i in range(10):
        print(a, end='')
        a += 1


print('Donne moi une valeur : ', end='')
a = int(input())
show_num(a)
